prow/config/jobs.go

```diff
@@ -103,9 +103,13 @@ type Brancher struct {
 	Branches []string `json:"branches"`
 }
 
+func (br Brancher) RunsAgainstAllBranch() bool {
+	return len(br.SkipBranches) == 0 && len(br.Branches) == 0
+}
+
 func (br Brancher) RunsAgainstBranch(branch string) bool {
 	// Favor SkipBranches over Branches
```

Comment:
stale comment?

Reply:
whoops, moved down for where this comment actually belongs to